README.md

```diff
@@ -12,6 +12,7 @@
 
 
 ## Authors
+- Collins Nana Antwi Baffour-Gyawu
 
 👤 **Author1**
 
```

Comment:
- [ ] Please since you are the only contributor to this project you can singularize the `Authors` section to `Author` and replace `Author1` with your name. :smile: